_posts/2000-01-01-intro.md

```diff
@@ -5,6 +5,8 @@ color: black
 style: center
 ---
 
+<img src="{{ site.logo}} " width="150px" height="150px">
```

Comment:
Is it possible to use css styling instead of inline styles ?

eg move width and height into css.

also spacing issue
should be:
`"{{ site.logo }}"`
instead of `"{{ site.logo}} "`